### DISCUSSION TITLE
Let discord show what your listening to

### DISCUSSION THREAD

**Discussion by corngoblin** [strawberrymusicplayer/strawberry#1235] [Ideas]
If you could change the name of the process that shows up in task manager or equivalent to what you're listening to, it'd show up in discord as your now playing.

You could also maybe add a different process some how that only changes it's name for discord to pick up.

I'm sure there are a bunch of ways to do this.. in linux it's easy enough, but I'm not sure about OSX and windows.

Just an idea Lol

**Comment by jonaski** [comment]
We have this now
